requirements.txt

```
@@ -1,4 +1,4 @@
-git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
+git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
```

We can just switch to the pypi version now. 
```suggestion
xblock-utils==1.2.2
```